storage/ndb/rest-server2/server/src/fs_cache.cpp

```diff
@@ -363,6 +363,14 @@ void FSMetadataCache::cache_entry_updater(Uint32 key_cache_id) {
         if (m_evicted || (milliSeconds >= eviction_ms)) {
           DEB_FS("FS Key %s deleted", first_entry->m_key.c_str());
           m_fs_cache[key_cache_id].erase(first_entry->m_key);
+          //unregister complex features from golang layer
+          printf("Cleaning unused entry\n");
```

Comment:
Do u really want to keep those printf's in production code, use e.g. DEB_FS instead